### ISSUE TITLE
Support for static media files

### ISSUE THREAD

**Issue by Nikhil-42** [CravingCrates/AnkiCollab-Plugin#14]
Anki supports using static media files in the `collection.media` folder. Files that are prefixed with an '_' are detected by Anki and even associated with the Decks that use them. My Deck relies on this feature to render the cards but AnkiCollab doesn't sync this file to the Google Drive folder.

**Comment by CravingCrates** [comment]
That's odd. Are they included if you try to export media files to disk? (In the Deck Overview click the gear icon next to your deck and AnkiCollab > Export Media to Disk)

**Comment by Nikhil-42** [comment]
No, using AnkiCollab > Export Media says "Exported 0 media files". 

Interestingly, exporting my Deck as an .apkg and checking "Include media" does include the appropriate static media.

**Comment by CravingCrates** [comment]
Gotcha. Thanks for the report. I will look into that.

**Comment by Nikhil-42** [comment]
This might be helpful https://github.com/ankitects/anki/blob/main/rslib/src/import_export/gather.rs#L145. In addition to gathering the media attached to each note, it also gathers the media stored in each notetype template.

**Comment by CravingCrates** [comment]
Thanks, yeah you're right. The media export is forked from this repo so I assume it has the same issue: https://github.com/abdnh/anki-media-exporter

Maybe abdo wants to look into it, too

**Comment by CravingCrates** [comment]
Hey, I'm currently working on a fix for this, but having troubles testing its functionality. Could you please upload a test deck that contains a card (and notetype) I can use to test my fix?

Thanks!
